Document Touch altitudeAngle, azimuthAngle, touchType properties (#43067)
mdn/content: files/en-us/web/api/touch/touchtype/index.md

---
title: "Touch: touchType property"
short-title: touchType
slug: Web/API/Touch/touchType
page-type: web-api-instance-property
browser-compat: api.Touch.touchType
---

{{ APIRef("Touch Events") }}

> The **`touchType`** read-only property of the {{domxref("Touch")}} interface returns the type of device that triggered the touch, indicating whether it was a direct touch from a finger or from a stylus.

Reviewer: I'm going to say "such as" because this might extend in future.

```suggestion
The **`touchType`** read-only property of the {{domxref("Touch")}} interface returns the type of device that triggered the touch, such as a stylus, or direct touch from a finger.
```
